tests/test_referenceObjectLoader.py

```diff
@@ -105,40 +101,58 @@ def testMakeMinimalSchema(self):
                     self.assertEqual("variable" in refSchema, addIsVariable)
                     self.assertEqual("photometric" in refSchema, addIsPhotometric)
                     self.assertEqual("photometric" in refSchema, addIsPhotometric)
-                    self.assertEqual("epoch" in refSchema, addProperMotion or addParallax)
                     self.assertEqual("coord_raErr" in refSchema, coordErrDim > 0)
                     self.assertEqual("coord_decErr" in refSchema, coordErrDim > 0)
                     self.assertEqual("coord_ra_dec_Cov" in refSchema, coordErrDim == 3)
-                    self.assertEqual("pm_ra" in refSchema, addProperMotion)
-                    self.assertEqual("pm_dec" in refSchema, addProperMotion)
-                    self.assertEqual("pm_flag" in refSchema, addProperMotion)
-                    self.assertEqual("parallax" in refSchema, addParallax)
-                    self.assertEqual("parallaxErr" in refSchema, addParallax)
-                    self.assertEqual("parallax_flag" in refSchema, addParallax)
+                    # We test setting thse these all together, below.
+                    self.assertNotIn("epoch", refSchema)
+                    self.assertNotIn("pm_ra", refSchema)
+                    self.assertNotIn("pm_dec", refSchema)
+                    self.assertNotIn("pm_flag", refSchema)
+                    self.assertNotIn("parallax", refSchema)
+                    self.assertNotIn("parallax_flag", refSchema)
```

Comment:
These should not be in the schema because the default for `addParallax` and `addProperMotion` is False, but I can remove this check if it seems confusing.